### ISSUE TITLE
Remove the notifier in Clair

### ISSUE THREAD

**Issue by reasonerjt** [goharbor/harbor#8064]
We used to use the notification handler to trigger rescan and make "scan_overview" up to date.

This cause some issue when the number of images in Harbor is large.

We should remove it after #8063 is done, as it's not needed for now.

**Comment by reasonerjt** [comment]
Fixed in #8466
